lms/envs/aws.py

```diff
@@ -673,16 +673,17 @@
 
 ##### Third-party auth options ################################################
 if FEATURES.get('ENABLE_THIRD_PARTY_AUTH'):
-    AUTHENTICATION_BACKENDS = (
-        ENV_TOKENS.get('THIRD_PARTY_AUTH_BACKENDS', [
-            'social_core.backends.google.GoogleOAuth2',
-            'social_core.backends.linkedin.LinkedinOAuth2',
-            'social_core.backends.facebook.FacebookOAuth2',
-            'social_core.backends.azuread.AzureADOAuth2',
-            'third_party_auth.saml.SAMLAuthBackend',
-            'third_party_auth.lti.LTIAuthBackend',
-        ]) + list(AUTHENTICATION_BACKENDS)
-    )
+    tmp_backends = ENV_TOKENS.get('THIRD_PARTY_AUTH_BACKENDS', [
```

Comment:
NIT: This is always returning  a list since it's reading that out of JSON so you don't need to do the casting below if you don't want to.